docs/dev/tutorials/aa-daily-spend-limit.md

```diff
@@ -915,7 +1029,7 @@ To make a transfer, run the following:
 yarn hardhat deploy-zksync --script deploy/transferETH.ts
 ```
 
-Although the error message doesn't give us any concrete reason, it's anticipated that the transaction was reverted like the below:
+Although the error message doesn't give us the specifics, we know it failed because the amount exceeded the limit. You should see following output:
 
 ```shell
```

Comment:
It might be a good idea to remove this output because it is nothing like what I saw.

Reply:
the error thrown is quite big but in some parts it actually mentions the mentioned output

Reply:
i couldn't see it